dev/import-beats/streams.go

```diff
@@ -75,10 +85,109 @@ func wrapVariablesWithDefault(mwvs manifestWithVars) manifestWithVars {
 	return withDefaults
 }
 
-func createMetricStreams() ([]util.Stream, error) {
+func createMetricStreams(modulePath, moduleName, datasetName string) ([]util.Stream, error) {
```

Comment:
Some function comments on these methods could be helpful to explain reviewers and future contributors, on what actually happens inside here.

Reply:
Fixed.